spec/serializers/search/article_serializer_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+RSpec.describe Search::ArticleSerializer do
+  let(:user) { create(:user) }
+  let(:organization) { create(:organization) }
+  let(:article) { create(:article, user: user, organization: organization) }
+
+  it "serializes an article" do
```

Comment:
These serializer specs are less about checking all the fields and more about making sure every field we ask for from an article is present and we dont raise an error trying to get it. I used this spec to figure out some fields that were actually only on podcasts and not on articles.